index.html

```diff
@@ -1,19 +1,27 @@
 ---
 permalink: /
 title: Introduction
-layout: default
+layout: none
```

Comment:
If you just remove `none` here then it will build. Currently this is looking for the layout "none"